<launch>
    <param name="/use_sim_time" value="true"/>

    <include ns="prueba" file="$(find auction)/launch/slam_gmapping.launch"/>
    <include ns="prueba" file="$(find auction)/launch/move_base_slam.launch"/>

    <node pkg="auction" type="goal_sender" name="goal_sender" ns="prueba"/>


</launch>
